core/pom.xml

```diff
@@ -137,6 +137,12 @@
             <artifactId>snakeyaml</artifactId>
             <version>${snakeyaml.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.spark</groupId>
+            <artifactId>spark-mllib_${scala.binary.version}</artifactId>
+            <version>${spark.version}</version>
+            <scope>${scope}</scope>
+        </dependency>
```

Comment:
With this new dependency, Is it included $SPARK_HOME/jars ? or it has to be added to the classpath when running the Qualification?

Reply:
I also wonder if this ML is standard across different platforms (databricks, EMR,..etc)?

Reply:
This dependency is added for unit tests to generate eventlogs consisting of ML funcs. We are not checking ML classes in the code instead it is string in stageInfo details where we are looking if there are any ML functions. 
`spark-mllib_2.12-{SPARK_VERSION}.jar`  is already included in $SPARK_HOME/jars. 
Current task was to look for eventlogs generated on Apache Spark as the current Spark-ML is not tested on CSP's yet. I will make it a note to get info if we need to test on eventlogs generated on Databricks, EMR etc. Thanks for noticing this.

Reply:
where is ${scope} set?  we want this ot just be test right?

Reply:
Correct. Updated the scope to be test.